code/modules/mob/living/carbon/human/human.dm

```diff
@@ -431,6 +431,38 @@
 				to_chat(usr, "<b>Notes:</b> [R.fields["notes"]]")
 				return
 
+			if(href_list["add_citation"])
+				if(!H.canUseHUD())
+					return
+				if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD))
+					return
+				var/t1 = stripped_input("Please input citation crime:", "Security HUD", "", null)
+				var/fine = FLOOR(input("Please input citation fine:", "Security HUD", 50) as num|null, 1)
+				if(isnull(fine))

```

Comment:
You need to check for the presence of the hud between and after the prompts. Also mention the max fine.

Reply:
this needs to be addressed

> You need to check for the presence of the hud between and after the prompts.